site/layouts/default.hbs

```diff
@@ -15,7 +15,12 @@
 		<main property="mainContentOfPage" class="col-md-9 col-md-push-3" typeof="WebPageElement">
 {{else}}
 <main property="mainContentOfPage" class="
-{{#unless page.noMainContainer}}container{{/unless}}
+{{#if page.noMainContainer}}
+{{else}}
+{{#isnt page.pageType "theme-topic"}}
```

Comment:
This extra logic is not needed, you can add the option "noMainContainer" set to "true" in the .hbs file configuration (in JSON at the top)

Reply:
Yep.  Sure.  I agree it is simpler.  However because "noMainContainer" is used in the dateModified component, I've had to modify its logic to not render the container class when pageType is theme-topic.  Done